.circleci/config.yml

```diff
@@ -0,0 +1,215 @@
+# iOS CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/ios-migrating-from-1-2/ for more details
+#
+version: 2
+jobs:
+
+  build:
+    working_directory: ~/code
+    docker:
+      - image: circleci/android:api-27-alpha
+    environment:
+      JVM_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - run:
+          name: build the whole project
+          command: bash gradlew build -x test
```

Comment:
Can we do a build with multiple android sdk versions? This is a newer version and might not catch any api compatibility issues.